start_bundle_coupled.R

```diff
@@ -117,8 +115,24 @@ if (length(argv) > 0) {
   if (! grep("scenario_config_coupled", path_settings_coupled))
     stop("Enter only a scenario_config_coupled* file via command line or set all files manually in start_bundle_coupled.R")
   path_settings_remind  <- sub("scenario_config_coupled", "scenario_config", path_settings_coupled)
+} else if (! file.exists(path_settings_coupled)) {
+  possiblecsv <- Sys.glob(c(file.path("./config/scenario_config_coupled*.csv"),
+                            file.path("./config","*","scenario_config_coupled*.csv")))
+  path_settings_coupled <- gms::chooseFromList(possiblecsv, type = "one coupled config file", returnBoolean = FALSE, multiple = FALSE)
+  path_settings_remind  <- sub("scenario_config_coupled", "scenario_config", path_settings_coupled)
+  message("")
 }
 
+message("path_remind:           ", if (dir.exists(path_remind)) green else red, path_remind, NC)
+message("path_magpie:           ", if (dir.exists(path_magpie)) green else red, path_magpie, NC)
+message("path_settings_coupled: ", if (file.exists(path_settings_coupled)) green else red, path_settings_coupled, NC)
+message("path_settings_remind:  ", if (file.exists(path_settings_remind)) green else red, path_settings_remind, NC)
+message("path_remind_oldruns:   ", if (dir.exists(path_remind_oldruns)) green else red, path_remind_oldruns, NC)
+message("path_magpie_oldruns:   ", if (dir.exists(path_remind_oldruns)) green else red, path_magpie_oldruns, NC)
```

Comment:
small error here: must be `path_magpie_oldruns` in the `dir.exists()` part. Is probably an old error…